dev/arktype.io/docs/api/morph.md

```diff
@@ -4,8 +4,19 @@ hide_table_of_contents: true
 
 # morph
 
-## text
+## operator
 
-```ts
-morph: Ark["morph"]
-```
+-   [morph](./morph.md)
+
+## tuple
+
+-   [inputType, &vert;&gt;, (data) =&gt; output]
+
+## helper
+
+-   morph(inputType, (data) =&gt; output)
+
+## example
+
+-   const morph = type( ["string", &vert;&gt; , (data) =&gt; `morphed ${data}`]) <br/>
```

Comment:
It seems like a lot of these problems come from the fact that it's hard to write TypeScript in JSDocs.

Maybe try an extension like this:

https://marketplace.visualstudio.com/items?itemName=galloween.jsdoc-highlight-code

So you'll know if you forget quotes around an operator, for example (don't worry about quoting T in the templates).

If that fails, just make sure you can copy the examples to a TS file without it yelling at you